tasks/e2e.sh

```diff
@@ -118,8 +119,9 @@ test -e build/static/media/*.svg
 test -e build/favicon.ico
 
 # Run tests
-npm run test
-test -e src/__tests__/__snapshots__/App-test.js.snap
+npm test -- --smoke-test
+# Uncomment when snapshot testing is enabled by default:
+# test -e src/__tests__/__snapshots__/App-test.js.snap
```

Comment:
Since it's disabled this doesn't really matter—when we re-add it we can revisit naming and where we put it.